docs/src/css/style.css

```diff
@@ -250,6 +256,14 @@ a.scroll-link:hover {
 .pt-card-wide {
   padding: 3rem 4.5rem 1rem;
 }
+@media screen and (min-width: 769px){
+  .column.is-10, .column.is-10-tablet{ 
```

Comment:
Instead of hardcoding media queries you can use [Bulma's mixins](https://bulma.io/documentation/overview/responsiveness/#breakpoints), and also you can avoid rewriting columns styles by using [responsive columns](https://bulma.io/documentation/columns/responsiveness/#different-column-sizes-per-breakpoint)

Reply:
commit ada8a2fd47e9225a8109c51f43f995d7b0b42f5e please refer to this commit's HTML file.